src/agent0/ethpy/base/transactions.py

```diff
@@ -21,6 +22,7 @@
 
 DEFAULT_READ_RETRY_COUNT = 5
 DEFAULT_WRITE_RETRY_COUNT = 1
+GAS_ESTIMATE_MULTIPLIER = 1.1
```

Comment:
we don't want this anymore, right?

Reply:
is it being used? is it better than nothing?

Reply:
I'm with Dylan, if we're adding an overwrite for gas, let's default to the rpc for the gas estimate without the multiplier

Reply:
removed in 6c05738bd101660aa9d51c745e3837906f098736